src/fmt.zig

```diff
@@ -2031,6 +2031,13 @@ const Formatter = struct {
 
                         return false;
                     },
+                    .suffix_single_question => |s| {
+                        if (fmt.nodeWillBeMultiline(AST.Expr.Idx, s.expr)) {
+                            return true;
+                        }
+
+                        return false;
```

Comment:
It seems like we can do `return fmt.nodeWillBeMultiline(AST.Expr.Idx, s.expr)` directly here?